gradle.properties

```diff
@@ -38,6 +38,7 @@ servletApiDependency=javax.servlet:javax.servlet-api:3.0.1
 portletApiDependency=org.apache.portals:portlet-api_2.1.0_spec:1.0
 
 # Dependency Versions
+activationVersion:1.0.2
```

Comment:
:question: is `activationVersion:1.0.2` intentional or was this meant to be `activationVersion=1.0.2`?